lib/libvcc/vcc_expr.c

```diff
@@ -146,13 +146,10 @@ vcc_DoubleVal(struct vcc *tl)
 void
 vcc_Duration(struct vcc *tl, double *d)
 {
-	double v, sc;
```

Comment:
Yes we should really drop the unit requirement, but accepting numbers as durations makes the syntax ambiguous. What if we kept the durations as-are and just folded `INT` into `DURATION`? IIUC this should be covered by your patch already and should achieve the same?